config/crd/bases/dynatrace.com_dynakubes.yaml

```diff
@@ -6616,6 +6624,147 @@ spec:
                           type: object
                         type: array
                     type: object
+                  logAgentDaemonSet:
```

Comment:
IN the ticket i see `LogAgentSpec` not logAgentDaemonSet. May be @chrismuellner  can say final word?

For example for EEC and OtelC we do use without Daemonset suffix, but it's Daemonset.

Reply:
I agree and removed the suffix. https://github.com/Dynatrace/dynatrace-operator/pull/3595/commits/d28e3b77f6b09a62c54494ddc435a1e7538c5ee5

If it is needed, I can readd it.